===================================================================
RCS file: /cvs/gnome/ooo-build/patches/src680/buildfix-longnames-54730.diff,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ooo-build/patches/src680/buildfix-longnames-54730.diff	2005/12/25 10:34:11	1.1
+++ ooo-build/patches/src680/buildfix-longnames-54730.diff	2005/12/25 10:41:23	1.2
@@ -0,0 +1,58 @@
+--- solenv/bin/modules/installer/languages.pm.all_lang	2005-09-08 05:04:40.000000000 -0400
++++ solenv/bin/modules/installer/languages.pm	2005-10-23 09:33:18.000000000 -0400
+@@ -64,6 +64,12 @@
+ 		$installer::globals::unixmultipath = $installer::globals::languagelist;
+ 		$installer::globals::unixmultipath =~ s/\,/\_/g;	# hashes not allowed, comma to underline
+ 		$installer::globals::alllanguagesinproductarrayref = installer::converter::convert_stringlist_into_array(\$installer::globals::unixmultipath, "_");
++
++		$installer::globals::unixmultipath_orig = $installer::globals::unixmultipath;
++		if (length($installer::globals::unixmultipath) > 120) {
++			chomp(my $shorter = `echo $installer::globals::unixmultipath | md5sum | sed -e "s/  -//g"` );
++			$installer::globals::unixmultipath = $shorter;
++		}
+ 	}	
+ 	
+ 	while ($first =~ /^(\S+)\#(\S+?)$/)	# Minimal matching, to keep the order of languages
+--- solenv/bin/modules/installer/globals.pm.all_lang	2005-10-20 14:49:34.000000000 -0400
++++ solenv/bin/modules/installer/globals.pm	2005-10-23 09:15:28.000000000 -0400
+@@ -124,6 +124,7 @@
+ 	$addpackagelist = "";
+ 	$is_unix_multi = 0;
+ 	$unixmultipath = "";
++	$unixmultipath_orig = "";
+ 	$alllanguagesinproductarrayref = "";
+ 	$shiptestdirectory = "";
+ 	$makelinuxlinkrpm = 0;
+--- solenv/bin/make_installer.pl.all_lang	2005-10-20 14:49:34.000000000 -0400
++++ solenv/bin/make_installer.pl	2005-10-23 09:31:02.000000000 -0400
+@@ -515,13 +515,29 @@
+ 	else { $logminor = $installer::globals::minor; }
+ 
+ 	my $loglanguagestring = $$languagestringref;
+-	if ( $installer::globals::is_unix_multi ) { $loglanguagestring = $installer::globals::unixmultipath; }
++	if ( $installer::globals::is_unix_multi ) { $loglanguagestring = $installer::globals::unixmultipath_orig; }
++
++	my $loglanguagestring_orig = $loglanguagestring;
++	if (length($loglanguagestring) > 120) {
++	    chomp(my $shorter = `echo $loglanguagestring | md5sum | sed -e "s/  -//g"`);
++	    $loglanguagestring = $shorter;
++	}
+ 
+ 	$installer::globals::logfilename = "log_" . $installer::globals::build . "_" . $logminor . "_" . $loglanguagestring . ".log";
+ 
+ 	if (( ! $installer::globals::is_unix_multi ) || ( $isfirstrun )) { $loggingdir = $loggingdir . $loglanguagestring . $installer::globals::separator; }
+ 
+ 	installer::systemactions::create_directory($loggingdir);
++
++	if ($loglanguagestring ne $loglanguagestring_orig) {
++	    (my $dir = $loggingdir) =~ s!/$!!;
++	    open(my $F1, "> $dir.dir");
++	    open(my $F2, "> " . $loggingdir . $installer::globals::logfilename . '.file');
++	    my @s = map { "$_\n" } split('_', $loglanguagestring_orig);
++	    print $F1 @s;
++	    print $F2 @s;
++	}
++
+ 	$installer::globals::exitlog = $loggingdir;
+ 
+ 	##############################################################
===================================================================
RCS file: /cvs/gnome/ooo-build/patches/src680/pld-splash.diff,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- ooo-build/patches/src680/pld-splash.diff	2005/10/27 20:22:23	1.1
+++ ooo-build/patches/src680/pld-splash.diff	2005/12/25 10:41:23	1.2
@@ -0,0 +1,43 @@
+Index: svx/prj/d.lst
+===================================================================
+RCS file: /cvs/graphics/svx/prj/d.lst,v
+retrieving revision 1.108
+diff -u -r1.108 d.lst
+--- svx/prj/d.lst	9 Dec 2004 16:36:45 -0000	1.108
++++ svx/prj/d.lst	16 Dec 2004 12:43:47 -0000
+@@ -55,7 +55,7 @@
+ ..\res\cjkintro.bmp %COMMON_DEST%\pck%_EXT%\starsuite_bitmap\intro.bmp
+ ..\res\ofaabout.bmp %COMMON_DEST%\pck%_EXT%\staroffice_bitmap\about.bmp
+ ..\res\soffice.bmp %COMMON_DEST%\pck%_EXT%\staroffice_bitmap\intro.bmp
+-..\res\ooointro.bmp %COMMON_DEST%\pck%_EXT%\openoffice\intro.bmp
++..\res\openintro_pld.bmp %COMMON_DEST%\pck%_EXT%\openoffice\intro.bmp
+ ..\res\ooonologointro.bmp %COMMON_DEST%\pck%_EXT%\openoffice\nologointro.bmp
+ 
+ mkdir: %_DEST%\inc%_EXT%\svx
+Index: svx/source/intro/ooo.src
+===================================================================
+RCS file: /cvs/graphics/svx/source/intro/ooo.src,v
+retrieving revision 1.11
+diff -u -r1.11 ooo.src
+--- svx/source/intro/ooo.src	7 Jan 2005 09:49:30 -0000	1.11
++++ svx/source/intro/ooo.src	1 Feb 2005 16:48:49 -0000
+@@ -48,8 +48,8 @@
+ #include "svxids.hrc"
+ #include "intro.hrc"
+ 
+-#ifdef BUILD_SPECIAL
+-#define OOO_VENDOR "Sun Microsystems Inc."
++#if 1
++#define OOO_VENDOR "PLD/Linux Team"
+ #else
+ //#define OOO_VENDOR "my company"
+ #endif
+@@ -56,7 +56,7 @@
+ 
+ Bitmap RID_DEFAULT_ABOUT_BMP_LOGO
+ {
+-	File = "openabout.bmp" ;
++	File = "openabout_pld.bmp" ;
+ };
+ 
+ String RID_APPTITLE
